assets/app/view/game/spreadsheet.rb

```diff
@@ -303,8 +303,12 @@ def render_corporation(corporation, operating_order, current_round, index)
         order_props = { style: { paddingLeft: '1.2em' } }
         operating_order_text = ''
         if operating_order.positive?
-          corporation.operating_history.each do |history|
-            operating_order_text = "#{operating_order}#{history[0] == current_round ? '*' : ''}"
+          if corporation.operated?
```

Comment:
let's change it to 

operating_order_text = 
if corporation.operated?
  "#{operating_order}{corporation.operating_history[-1][0] == current_round ? "*" : ""}"
else
  operating_order.to_s
end

Reply:
`corporation.operating_history` is a hash, so I don't think `corporation.operating_history[-1]` works (at least, it doesn't for me).  

`corporation.operating_history.keys[-1] == current_round`

does do the right thing for me, and `keys[-1]` is defined but nil for empty hashes / non-operated companies, so it's possible to replace the whole conditional with one line:

`operating_order_text = "#{operating_order}#{corporation.operating_history.keys[-1] == current_round ? '*' : ''}"`

For non-operated companies, `nil != current_round`, and we do the right thing (get the operating_order number, but don't append the '*').

Is that ok?